homeassistant/components/demo/climate.py

```diff
@@ -258,6 +259,8 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
         ):
             self._target_temperature_high = kwargs.get(ATTR_TARGET_TEMP_HIGH)
             self._target_temperature_low = kwargs.get(ATTR_TARGET_TEMP_LOW)
+        if kwargs.get(ATTR_HVAC_MODE) is not None:
```

Comment:
Please use a walrus expression to store the looked up value in a local variable that we can use on the next line.